frontend/src/hooks/api/auditLogs/queries.tsx

```diff
@@ -5,27 +5,29 @@ import { apiRequest } from "@app/config/request";
 import { Actor, AuditLog, AuditLogFilters } from "./types";
 
 export const workspaceKeys = {
-  getAuditLogs: (workspaceId: string, filters: AuditLogFilters) =>
+  getAuditLogs: (filters: AuditLogFilters, workspaceId: string | null) =>
     [{ workspaceId, filters }, "audit-logs"] as const,
   getAuditLogActorFilterOpts: (workspaceId: string) =>
     [{ workspaceId }, "audit-log-actor-filters"] as const
 };
 
-export const useGetAuditLogs = (workspaceId: string, filters: AuditLogFilters) => {
+export const useGetAuditLogs = (filters: AuditLogFilters, workspaceId: string | null) => {
   return useInfiniteQuery({
-    queryKey: workspaceKeys.getAuditLogs(workspaceId, filters),
+    queryKey: workspaceKeys.getAuditLogs(filters, workspaceId),
+    enabled: workspaceId !== "",
+
     queryFn: async ({ pageParam }) => {
-      const { data } = await apiRequest.get<{ auditLogs: AuditLog[] }>(
-        `/api/v1/workspace/${workspaceId}/audit-logs`,
-        {
-          params: {
-            ...filters,
-            offset: pageParam,
-            startDate: filters?.startDate?.toISOString(),
-            endDate: filters?.endDate?.toISOString()
-          }
+      const auditLogEndpoint = workspaceId
+        ? `/api/v1/workspace/${workspaceId}/audit-logs`
+        : "/api/v1/organization/audit-logs";
```

Comment:
Note: In the future as we move audit logs out of projects into the organization level, I would deprecate `/v1/workspace/:workspaceId/audit-logs` and add `projectId` as a query param on the corresponding organization level endpoint for audit logs.

No action needed; just comment.